Makefile

```diff
@@ -81,11 +81,21 @@ release: ebpf
 	$(call build, build --release, building retis (release))
 
 test: ebpf
-	$(call build, test, building and running tests)
+ifeq ($(COV),1)
+	$(CARGO) llvm-cov clean --workspace -q
+	$(call build,llvm-cov $(if $(VERBOSITY),,-q),running tests with coverage)
+else
+	$(call build,test,building and running tests)
+endif
 
 bench: ebpf
 	$(call build, build -F benchmark --release, building benchmarks)
 
+ifeq ($(COV),1)
+report:
+	$(CARGO) llvm-cov report --html
```

Comment:
IMO, this could be defined regardless of the value of `$(COV)` so my suggestion is to remove the `ifeq`, WDYT?

Furthermore, I would replace `--html` with `$(CARGO_CMD_OPTS)` or something equivalent. Mostly because it would be useful to do things like:
```sh
CARGO_OPTS=+stable make report COV=1 CARGO_CMD_OPTS="--lcov --output-path retis.lcov"
```

or without parameters show the result on the console.